dnf-docker-test/features/protected_packages.feature

```diff
@@ -7,31 +7,31 @@ Scenario: Removal of installed protected package
    | State        | Packages      |
    | installed    | TestA, TestB  |
 
- When I execute "dnf" command "-y remove TestA --setopt=protected_packages=TestA" with "failure"
+ When I execute "dnf" command "-y remove TestA --setopt=protected_packages=TestA" with "fail"
  Then transaction changes are as follows
    | State        | Packages     |
    | absent       | TestA, TestB |
```

Comment:
| present | TestA, TestB |

The packages were installed in step above, and not changed be previous step, therefore they are still present. Absent means that they are not installed, but they wer not installed even before last dnf command.